go/vt/vtgate/executor_select_test.go

```diff
@@ -1137,7 +1137,7 @@ func TestSelectBindvars(t *testing.T) {
 
 	// When there are no matching rows in the vindex, vtgate still needs the field info
 	wantQueries = []*querypb.BoundQuery{{
-		Sql: "select id from `user` where 1 != 1",
+		Sql: "select id from `user` where `name` = :name",
```

Comment:
I don't understand why we send that where clause for the field query, can't it just be `1 != 1`?